runtime/src/start.rs

```diff
@@ -69,33 +76,41 @@ pub async fn start(loader: impl Loader + Send + 'static, runner: impl Runner + S
     };
 
     // this is handled after arg parsing to not interfere with --version above
-    #[cfg(feature = "setup-tracing")]
+    #[cfg(all(feature = "setup-tracing", not(feature = "setup-telemetry")))]
     {
         use tracing_subscriber::{fmt, prelude::*, registry, EnvFilter};
         registry()
             .with(fmt::layer().without_time())
             .with(
                 // let user override RUST_LOG in local run if they want to
-                EnvFilter::try_from_default_env()
+                EnvFilter::try_from_default_env().unwrap_or_else(|_| {
                     // otherwise use our default
-                    .or_else(|_| {
-                        EnvFilter::try_new(if args.beta {
+                    Into::into(format!(
+                        "{},{}=debug",
```

Comment:
Changing the log level for user crates is good to note down in the PR so we remember it in release notes.

Reply:
Added to PR description 🙂